### PR TITLE
created AdminReviewRepo with defined methods, created IAdminReviewRep…

### LINKED ISSUE #88: Replace Fake Admin Review Repository
DOD: When there is a new Admin Review Repository that connects all of the current functionality to the Database.

### PR DESCRIPTION
**Pull request by taerichard**
…o that contains the signature only

### CODE REVIEW

**Comment by taerichard** [comment]
This is the IAdminReviewRepo file that will be used in the controllers.    

    // knows about the dbcontext issue 88
    public interface IAdminReviewRepository 
    {
        IEnumerable<Capture> GetCaptures();
        Capture GetCapture(long captureId);
        IEnumerable<Capture> GetAllUnapproved();
        Capture UpdateCapture(Capture capture);
        Capture DeleteCapture(long captureId);
        void SaveChanges(Capture capt);  // from previous classs?  
    }
`

File: photo_points/Repositories/AdminReviewRepository.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            return capture;
+        }
+
+        public IEnumerable<Capture> GetCaptures()
+        {
+            var captures = _dbcontext.Captures;
+            return captures;
+        }
+
+        public void SaveChanges(Capture capt)
```
Comment (taerichard) [inline photo_points/Repositories/AdminReviewRepository.cs]: this is the update part after fixing the details of the capture

File: photo_points/Repositories/AdminReviewRepository.cs [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            _dbcontext = Dbcontext;
+        }
+        public Capture DeleteCapture(long captureId)
+        {
+            // find capture to delete
+            var captureToDelete = _dbcontext.Captures.FirstOrDefault(p => p.captureID == captureId);
+
+            if(captureToDelete != null)
+            {
+                _dbcontext.Captures.Remove(captureToDelete);
```
Comment (taerichard) [inline photo_points/Repositories/AdminReviewRepository.cs:24]: + save changes

**Review by floydfix — CHANGES_REQUESTED** [review]
Does not build. Error in the new AdminReviewRepository. When you fix that, remove the FakeAdminReviewRepository and change any remaining calls from it to your new AdminReviewRepository, and make sure it all still works

**Comment by taerichard** [comment]
No error with build now.  Created Test Capture action in admin controller to check if it works.  Seems fine, just need capture data populated.

File: photo_points/Controllers/AdminController.cs [hunk 3]
```diff
@@ -17,32 +17,28 @@ namespace photo_points.Controllers
     public class AdminController : Controller
     {
 
-        private IAdminReviewServices _adminReviewServices;
-        private PhotoDataContext _dbc;
-
-        public AdminController(IAdminReviewServices adminServiceReview, PhotoDataContext dbc)
```
Comment (gjesionowski) [inline photo_points/Controllers/AdminController.cs:23]: The second part of the old line 23 "public AdminController(IAdminReviewServices adminServiceReview, PhotoDataContext dbc)" being removed seems to cause an issue with changing to views that use the database. I mean the ", PhotoDataContext dbc" part.

**Review by gjesionowski — CHANGES_REQUESTED** [review]

File: photo_points/Controllers/AdminController.cs (hunk 3 above)
Comment (gjesionowski) [reply photo_points/Controllers/AdminController.cs:23]: Yah tested on my machine and replacing the new line 22 with this should fix it
`AdminController(IAdminReviewRepository adminReviewRepo, PhotoDataContext dbc)
        {
            _adminReviewRepo = adminReviewRepo;
            _dbc = dbc;
        }`

**Pull request closed without merging**